README.md

```diff
@@ -85,11 +85,11 @@ const options = {
   grabbedNode: node => true, // filter function to specify which nodes are valid to grab and drop into other nodes
   dropTarget: (dropTarget, grabbedNode) => true, // filter function to specify which parent nodes are valid drop targets
   dropSibling: (dropSibling, grabbedNode) => true, // filter function to specify which orphan nodes are valid drop siblings
-  newParentNode: (grabbedNode, dropSibling) => ({}), // specifies element json for parent nodes added by dropping an orphan node on another orphan (a drop sibling)
   boundingBoxOptions: { // same as https://js.cytoscape.org/#eles.boundingBox, used when calculating if one node is dragged over another
     includeOverlays: false,
     includeLabels: true
   },
+  newParentNode: (grabbedNode, dropSibling) => ({}), // specifies element json for parent nodes added by dropping an orphan node on another orphan (a drop sibling). You can chose to return the dropSibling in which case it becomes the parent node and will be preserved after all its children are removed. 
```

Comment:
It looks like this got moved in the merge.  It should be beside the other functions, like dropSibling